plugins/VersionUpgrade/VersionUpgrade33to34/VersionUpgrade33to34.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2018 Ultimaker B.V.
+# Cura is released under the terms of the LGPLv3 or higher.
+
+import configparser #To parse preference files.
+import io #To serialise the preference files afterwards.
+
+from UM.VersionUpgrade import VersionUpgrade #We're inheriting from this.
+
+
+##  Upgrades configurations from the state they were in at version 3.2 to the
+#   state they should be in at version 3.3.
+class VersionUpgrade33to34(VersionUpgrade):
+
+    ##  Gets the version number from a CFG file in Uranium's 3.2 format.
+    #
+    #   Since the format may change, this is implemented for the 3.2 format only
+    #   and needs to be included in the version upgrade system rather than
+    #   globally in Uranium.
```

Comment:
More copy-pasted documentation is wrong.

Reply:
Fixed in ad3317a99